src/themes/theme.tsx

```diff
@@ -0,0 +1,20 @@
+import { createTheme } from '@mui/material';
+
+const theme = createTheme({
+  palette: {
+    primary: {
+      main: '#16423C'
```

Comment:
Use an enum called `Colors` instead of hard-coded values.
Name them with either a descriptive value or what you're using it here as.
e.g. `darkGreen = '#16423C'` or `main = '#16423C'`
Whatever sounds better to you.